mpd/mpd_test.go

```diff
@@ -299,7 +299,7 @@ func TestAddNewContentProtectionRootErrorEmptyDefaultKID(t *testing.T) {
 	m := NewMPD(DASH_PROFILE_LIVE, VALID_MEDIA_PRESENTATION_DURATION, VALID_MIN_BUFFER_TIME)
 	s, _ := m.AddNewAdaptationSetVideo("7357", DASH_MIME_TYPE_VIDEO_MP4, VALID_SCAN_TYPE, VALID_SEGMENT_ALIGNMENT, VALID_START_WITH_SAP)
 
-	cp, err := s.AddNewContentProtectionRoot("")
+	cp, err := s.AddNewContentProtectionRoot2("")
```

Comment:
Can we come up with a more descriptive name please?

Reply:
AddNewContentProtectionRootThatDoesntMisformatUUIDs

Reply:
`AddNewContentProtectionRootFixedUUID`?